build.md

```diff
@@ -0,0 +1,139 @@
+# Amplifier Build Instructions
+
+This document provides instructions for building Amplifier components, including contracts and the `ampd` service.
+
+## Prerequisites
+
+- Rust and Cargo (rust-version = "1.81.0" currently required)
+- Docker (for contract optimization)
+- Git (for cloning and handling submodules)
+- cargo-run-script (for running optimization scripts)
+
+## Initial Setup
+
+When cloning the repository, make sure to initialize and update submodules:
+
+```bash
+# Clone the repository
+git clone https://github.com/axelarnetwork/axelar-amplifier.git
```

Comment:
```suggestion
git clone --recurse-submodules  https://github.com/axelarnetwork/axelar-amplifier.git
```
easier setup

Reply:
this kind of crucial setup cmd should be shown in the readme itself to reduce the risk of missing it and causing errors